DEVELOPER_GUIDE.md

```diff
@@ -43,7 +43,7 @@ To develop on OpenSearch Dashboards, you'll need:
 
 - A [GitHub account](https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account)
 - [`git`](https://git-scm.com/) for version control
-- [`Node.js`](https://nodejs.org/), [`npm`](https://www.npmjs.com/), and [`Yarn`](https://yarnpkg.com/) for building and running the project
+- [`Node.js`](https://nodejs.org/), [`npm`](https://www.npmjs.com/)( Note that, for Darwin (MacOS) only, it is important to have the latest version of Node.js in your machine, in other case you will have an issue with [`grunt`]plugin ), and [`Yarn`](https://yarnpkg.com/) for building and running the project
```

Comment:
Are you saying here that you need a version _newer_ than the version in `.nvmrc`? Or just specifying that the version of node installed is important (which it is). See the note on L60 for the source of truth about the expected node version.

Reply:
Hi, thank you for your response, let me explain my situation

There was a problem when I followed our developer's guide. I had initially node installed in my MacOS. After installing nvm 14.20.1, I faced one problem, specifically package incompatibility(I will attach my screenshots below). 

Error: grunt@1.6.1. The engine node is incompatible with this module. Expected version >=16 Got 14.20.1. 

However, when I switched to the 16 th version of nvm, I had package incompatibility with opensearch-dashboard@3.0.0. So, this was very strange for me. How did I resolved this isssue?
I wrote on my laptop's terminal:
aigerimsuleimenova@Aigerims-MacBook-Air ~ % npm install n -g
added 1 package in 1s,
 Doing this before, I installed node by  npm install.
This command added one package, which other version on npm can not provide.